### ISSUE TITLE
Is this sufficiently robust for long-term and/or large scale use?

### ISSUE THREAD

**Issue by SteveMacenski** [NVIDIA-ISAAC-ROS/isaac_ros_visual_slam#58]
If so, I'd be happy to have some documentation about working with Nav2 + VSLAM in our tutorials: https://navigation.ros.org/tutorials/index.html. However, its important that its suitable for Nav2 users for practical applications!

**Comment by hemalshahNV** [comment]
We're racking up mileage on Isaac ROS Visual SLAM but if there is feedback on its performance in large-scale use for long-terms, it would be greatly appreciated. It should be suitable for Nav2 users for practical applications but I'd rather have Nav2 users confirm this of course.

**Comment by SteveMacenski** [comment]
I don't have any in particular to share, but would love to hear as well as comparison of this to other methods. I'd be happy to have it as a tutorial or integration into ROS 2 as an alternative option for SLAM/localization using vision if its sufficiently good for practical use.

**Comment by hemalshahNV** [comment]
Yes, Isaac ROS Visual SLAM does have multi-session mapping (save a map, restore, continue mapping). It would be great to work out a series of benchmarks together to qualify both navigation behavior and subsystems including localization, floor plan mapping, trajectory generation, etc. if there isn't already one you would recommend (continuing the discussion from a ROS Discourse thread).
